packages/manager/apps/cloud/client/app/dbaas/logs/detail/logs-detail.less

```diff
@@ -15,6 +15,19 @@
   }
 }
 
+ul.overview {
```

Comment:
The `ul` HTML element already provide this behavior.
The issue here is that the padding left is removed via the following helper: `pl-0`.
Maybe you can simply remove it. Is that make sense for you?

In that way, you will be able to get rid of all the styles added here.

Reply:
You are right, additional style is not useful here. I removed it.